docs/source/ProgrammingGuide/View.rst

```diff
@@ -145,7 +145,57 @@ Another issue is that View construction in a Kokkos parallel region does not upd
 
 Here is how to create a View of Views, where each inner View has a separate owning allocation:
 
-1. The outer View must have a memory space that is both host and device accessible, such as `CudaUVMSpace`.
+1. The outer View must have a memory space that is both host and device accessible, such as :cppkokkos:type:`SharedSpace`.
```

Comment:
Should the possibility of using a `DualView` for the outer view be mentioned as an alternative?